### ISSUE TITLE
[rush] Coloring eventHook fail

### ISSUE THREAD

**Issue by VincentSurelle** [microsoft/rushstack#1745]
## Please prefix the issue title with the project name i.e. [rush], [api-extractor] etc. ##

**Is this a feature or a bug?**

- [x] Feature
- [ ] Bug

**Please describe the actual behavior.**

I would like to colorize this kind of message :
```
Event hook “node common/scripts/post-rush-install.js” failed. Run “rush” with --debug to see detailed error information
```

It's an error than can have impact on some projects. I think keeping these messages white can lead to unexpected errors.

Red can be too much for some projects. I think yellowing the message to make it a warning is good enough.

I can obviously make a PR for this ;)

I just want to know if : 

- It's ok ?
- Red or Yellow ?

Thanks

**Comment by iclanton** [comment]
>It's ok ?

Go ahead!

>Red or Yellow ?

I agree with you that red may be too much. Yellow sounds good.

**Comment by VincentSurelle** [comment]
Merged in #1747
